.eslintrc.js

```diff
@@ -82,6 +82,7 @@ module.exports = {
     'react/react-in-jsx-scope': 2, // Prevent missing React when using JSX
     'react/self-closing-comp': 2, // Prevent extra closing tags for components without children,
     'react/prefer-stateless-function': 2, // Use functional components vs classes
+    'react/display-name': 0,
```

Comment:
This rule is sort of helpful for viewing the dom tree while debugging something and for error outputs, to remove the errors, we would have to name the functions and not resort to anonymous declarations, I found that it is annoying sometimes but it encourages to properly name components. Definitely not important but to remove the error on the header declarations we could use a named function or pass the `CloseButton` or `BackButton` functions directly as the value I think. totally up to you but we could leave the rule if we want to.

Reply:
I definitely agree with your points here!  Ill make the updates to allow to add this back in.